modules/catalog-magento/mappers/ToProduct.ts

```diff
@@ -8,6 +8,7 @@ import ToGroupedItem from '#ioc/mappers/ToGroupedItem'
 
 export default (data: any) => ({
   __typename: data.__typename ?? '',
+  brand: (data.brand ?? '') as string,
```

Comment:
I would expect brand to be at least object with a string called name. Even if pure Magento only returns string, we need to be prepared for other integrations that might also send additional stuff.

Reply:
I created object within brand with name property. Please, check it now.